bookwyrm/templates/user/layout_without_bio.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'layout.html' %}
+{% load i18n %}
+{% load humanize %}
+{% load utilities %}
+{% load markdown %}
+{% load layout %}
+
+{% block title %}{{ user.display_name }}{% endblock %}
+
+{% block opengraph_images %}
+    {% include 'snippets/opengraph_images.html' with image=user.preview_image %}
+{% endblock %}
+
+{% block content %}
+<header class="block">
+    {% block header %}
+    <h1 class="title">
+        {% trans "User Profile" %}
+    </h1>
+    {% endblock %}
+</header>
+
+
+{% block tabs %}
```

Comment:
Since this layout is only being used by one template, I think it makes more sense to have `bookwyrm/templates/user/shelf/shelf.html` inherit directly from `templates/layout.html` and remove this entirely.

Reply:
Hi if we directly use templates/layout.html then tabs All books, To Read, Currently Reading, Read are no displayed please find screenshot below.


![image](https://user-images.githubusercontent.com/44121606/123991105-0980ad00-d9e8-11eb-9036-fc64c2f24801.png)

Reply:
Sorry, I should have been more specific. The child template has to call the blocks that are defined in the parent template. In the user layout template, the two important blocks are the `tabs` block, which has a default set of tabs that can be overwritten, and the `panel` block for the main body of the page. This layout uses a grandparent layout (sorry, it's so complicated!), `templates/layout.html` that has a `content` block, which contains the tabs and panel blocks.

To use `templates/layout.html` directly, you have to call the `content` block, and include the html that is currently wrapped in the `tabs` and the `panel` in that block. 

Does that make any sense at all?